spp_import_match/models/import_match.py

```diff
@@ -0,0 +1,127 @@
+# Part of OpenSPP. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, tools
+
+
+class SPPImportMatch(models.Model):
+    _name = "spp.import.match"
+    _description = "Import Matching"
+    _order = "sequence, name"
+
+    name = fields.Char(compute="_compute_name", store=True, index=True)
+    sequence = fields.Integer(index=True)
+    model_id = fields.Many2one(
```

Comment:
I think there should be an unique constraint for model_id in spp import match, to narrow down the searching, and give us back better performance